lib/Tumblr/StreamBuilder/Codec/Codec.php

```diff
@@ -81,11 +81,11 @@ public function __construct(CacheProvider $cache_provider = null)
     }
 
     /**
-     * Encode a templatable object.
-     * @param Templatable $obj The object to encode.
+     * Encode an object.
+     * @param mixed $obj The object to encode.
```

Comment:
```suggestion
     * @param Templatable|string $obj The object to encode.
```
Since we know anything else will cause an exception.